parsl/configs/beagle_single_node.py

```diff
@@ -34,7 +34,8 @@
                 init_blocks=1,
                 max_blocks=1,
                 launcher=AprunLauncher,
-                overrides='OVERRIDES',     # Please replace OVERRIDES with your overrides
+                scheduler_options='SCHEDULER_OPTIONS',     # Please replace SCHEDULER_OPTIONS with your scheduler_options
```

Comment:
Sorry I didn't catch this in the previous PR, but maybe as an example this should just be an empty string? So that the example would be closer to something that works out-of-the-box, since there may be cases where one does not need to add their own scheduler options. Same with`worker_init`. I don't have a strong opinion about this though, feel free to ignore.